python/lsst/meas/extensions/ngmix/LMSimpleShape.py

```diff
@@ -206,19 +207,23 @@ def measure(self, measRecord, exposure):
                     SingleFrameLMSimpleShapePlugin.ErrEnum.flag_singularMatrix)
             if res['flags'] & LM_NEG_COV_EIG:
                 raise MeasurementError(
-                    self.flagHandler.getDefinition(SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig).doc,
+                    self.flagHandler.getDefinition(
+                        SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig).doc,
```

Comment:
A single line is 135 characters. I could change it to 
```python
raise MeasurementError(self.flagHandler.getDefinition(
                       SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig).doc,
                       SingleFrameLMSimpleShapePlugin.ErrEnum.flag_negativeCovEig)
```
which looks a little better but could be more confusing. I don't mind changing it though.